src/Components/Todos/Todos.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  key: number;
```

Comment:
As a best practice, using the `key` prop within the component is not recommended. The `key` prop is used by React to track and manage components within a list of components. Please remove the `key` prop from the `Props` type, and add the key when you map over the components in the parent component. 🔑